src/utilities.jl

```diff
@@ -262,12 +343,11 @@ function printmethod(buffer::IOBuffer, binding::Docs.Binding, func, method::Meth
         join(buffer, kws, ", ")
     end
     print(buffer, ")")
-    if typesig isa UnionAll
-        t = typesig.body.a
-    else
-        t = typesig
+    for t in reverse(where_syntax)
+        s = typevar_to_string(t)
```

Comment:
It seems that this is the only place where `typevar_to_string` is used. Would it make sense to do this instead and get rid of that function?

```suggestion
        s = "where $t" # t is a TypeVar
```